### PR TITLE
NAS-109712 / 21.04 / Use updated endpoints to manipulate VM display/vnc devices in DeviceAddComponent, DeviceEditComponent, VmListComponent

### PR DESCRIPTION
**Pull request by ryousaf**
NAS-109712

### CODE REVIEW

File: src/app/helptext/vm/devices/device-add-edit.ts [hunk 1]
```diff
@@ -85,17 +85,17 @@ rootpwd_tooltip : T('Enter a password for the <i>rancher</i> user. This\
 boot_placeholder : T('boot'),
 boot_tooltip : '',
 
-vnc_port_placeholder : T('Port'),
-vnc_port_tooltip : T('Can be set to <i>0</i>, left empty for FreeNAS to\
+port_placeholder : T('Port'),
+port_tooltip : T('Can be set to <i>0</i>, left empty for FreeNAS to\
```
Comment (stavros-k) [inline src/app/helptext/vm/devices/device-add-edit.ts:89]: Just happened to watch some PRs. Shouldn't this be `TrueNAS` instead of `FreeNAS`?

Comment (ryousaf) [reply src/app/helptext/vm/devices/device-add-edit.ts:89]: Yes, you are correct. Thanks for the review! Will update.

**Review by denysbutenko — COMMENTED** [review]
PR looks good, still waiting for string update

**Comment by ryousaf** [comment]
> PR looks good, still waiting for string update

Updated

**Review by denysbutenko — APPROVED** [review]

**Pull request merged**